ADMVEDU95/ADMVEDU95/Extensions/UIView+Anchor.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIView+Anchor.swift
+//  ADMVEDU95
+//
+//  Created by Satsishur on 19.04.2021.
+//
+
+import UIKit
+
+extension UIView {
+    func anchor(top: NSLayoutYAxisAnchor? = nil, paddingTop: CGFloat = 0,
```

Comment:
let's create separate methods where we don't put all parameter but only that we need to set layout